template-helpers/breaches.js

```diff
@@ -32,70 +30,28 @@ function dataClassesforCards(breach, locales) {
   return localizedBreachDataClasses(topTwoClasses.slice(0, 2), locales);
 }
 
-function makeBreachCards(breaches, locales, filter=true) {
-  const breachCardStrings = getLocalizedBreachCardStrings(locales);
-  const formattedBreaches = [];
-
-
-  if (filter && breaches.length > 1) {
-    breaches.sort((a,b) => {
-      const oldestBreach = new Date(a.BreachDate);
-      const newestBreach = new Date(b.BreachDate);
-      return newestBreach-oldestBreach;
-    });
-  }
-
-  for (const breach of breaches) {
-    const breachCard = JSON.parse(JSON.stringify(breach));
-    const breachCategory = getBreachCategory(breach);
+function sortBreaches(breaches) {
+  breaches = breaches.sort((a,b) => {
+    const oldestBreach = new Date(a.BreachDate);
+    const newestBreach = new Date(b.BreachDate);
+    return newestBreach-oldestBreach;
+  });
+  return breaches;
+}
 
-    breachCard.AddedDate = prettyDate(breach.AddedDate, locales);
-    breachCard.PwnCount = localeString(breach.PwnCount,locales);
-    breachCard.DataClasses = dataClassesforCards(breach, locales);
+function makeBreachCards(breaches, locales) {
+  const formattedBreaches = [];
+  const breachCardStrings = getLocalizedBreachCardStrings(locales);
+  breaches = JSON.parse(JSON.stringify(breaches));
```

Comment:
Do you have to `JSON.parse(JSON.stringify())`? Can you just use `breaches` without `stringify` and `parse`?

Reply:
There may be a way, but after lots of trying different approaches and stack overflowing I wasn't able to find it.